lib/models/release.coffee

```diff
@@ -389,6 +409,12 @@ getReleaseModel = (deps, opts) ->
 			$expand:
 				release_tag:
 					mergePineOptions
+						$select: [
```

Comment:
Shall we extract this in a variable for consistency?

Reply:
Sure :+1: